platform/feature-api/api/src/pod-api/uri.ts

```diff
@@ -9,8 +9,8 @@ export function createUUID(): string {
     return uuidv4();
 }
 
-const polyProtocol = "polypod://";
-const polyProtocolRegex = new RegExp(`^${polyProtocol}`);
+export const polyProtocol = "polypod://";
+export const polyProtocolRegex = new RegExp(`^${polyProtocol}`);
```

Comment:
See above - I'm not a fan of exposing things for test purposes, the ability to mock components not under test being the only exception. Otherwise, things should be as private as possible and tests should make an effort to test code paths through the public API.

Reply:
Not really for testing; maybe I should have factored out this change as I have done with the rest. Main intention here is to make the polyProtocol a single source for all the literals that are used all over.

Reply:
Just realised that you could just call isPolypodUri with no need to export these internals.

That said, I'm still not sure why the unit tests for poly-import need to ensure that importArchive works as expected... They can't even, only in the test feature can we test API across platforms. The poly-import tests are for testing code in this library, nothing outside it.